NEWS.md

```diff
@@ -37,6 +37,8 @@ New library functions
 Standard library changes
 ------------------------
 
+* When `wait` (or `@sync` etc.) is called on a failing `Task`, the exception is propagated as a
+  `FailedTaskException` wrapping the task ([#32814]).
```

Comment:
You might want a few words about why this is an improvement (i.e. you no longer lose the backtrace).

Reply:
(should also mention `fetch` in the docs)